docker-compose.yml

```diff
@@ -6,10 +6,18 @@ services:
       - postgres_data:/var/lib/postgresql/data
     environment:
       - POSTGRES_PASSWORD=password
+    #when upgrading elasticsearch server on local machine:
+    #network_mode: "host"
+    ports:
+      - "5432:5432"
   elastic:
-    image: elasticsearch:2.4.5
-    volumes:
-      - elastic_data:/usr/share/elasticsearch/data
+    image: elasticsearch:6.7.2
+    # we commented out the next 2 lines so version 6 could work
+    # TODO: find out if it's needed / how to make it work
+    # volumes:
+    #   - elastic_data:/usr/share/elasticsearch/data
```

Comment:
Is there still anything to check? Or can we delete these comments as well?

Reply:
we still need to check. the volumes are not a must but a nice to have, so one won't have to recreate the index every time they delete the container